JaeWook/week1/BOJ1929.js

```diff
@@ -0,0 +1,15 @@
+let fs = require("fs");
+let input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");
+
+const [n, m] = input
+  .shift()
+  .split(" ")
+  .map((e) => parseInt(e));
```

Comment:
이거 있자나요 
```suggestion
  .map((v) => +v);
```
이렇게 하면 더 편하다요?

Reply:
요거 근데 .map(Number)이렇게 해도 되더라고?!